### PR TITLE
W800rf32

### PR DESCRIPTION
**Pull request by horga83**
**Description:**
Initial add of documentation for w800rf32 component and binary_sensor platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17920

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/w800rf32.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+```yaml
+# Example configuration.yaml entry
+w800rf32:
+  device: PATH_TO_DEVICE
+```
+
+Configuration variables:
+
+- **device** (*Required*): The path to your device, e.g., `/dev/ttyUSB0`
```
Comment (klaasnicolaas) [inline source/_components/w800rf32.markdown]: Same

File: source/_components/binary_sensor.w800rf32.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        off_delay: 5
+        device_class: Motion
+      a2:
+        name: motion_kitchen
+        device_class: Motion
+```
+
+Configuration variables:
+
+- **device_class** (*Optional*): The [type or class of the sensor](/components/binary_sensor/) to set the icon in the frontend.
```
Comment (klaasnicolaas) [inline source/_components/binary_sensor.w800rf32.markdown]: Please use the new configuration format, see [here](https://developers.home-assistant.io/docs/en/documentation_create_page.html) for meer info.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/w800rf32.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+```yaml
+# Example configuration.yaml entry
+w800rf32:
+  device: PATH_TO_DEVICE
+```
+
+{% configuration %}
+device:
+  description: The path to USB/serial device, example: /dev/ttyUSB0.
```
Comment (klaasnicolaas) [inline source/_components/w800rf32.markdown]:
```suggestion
  description: "The path to USB/serial device, example: /dev/ttyUSB0."
```

File: source/_components/binary_sensor.w800rf32.markdown [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+title: "W800rf32 Binary Sensor"
+description: "Instructions on how to integrate W800rf32 binary sensors into Home Assistant."
+date: 2018-10-16 12:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: w800rf32.png
+ha_category: Binary Sensor
+ha_release: 0.79
```
Comment (klaasnicolaas) [inline source/_components/binary_sensor.w800rf32.markdown]: Pherhaps release version must be updated to 0.83, if this [PR](https://github.com/home-assistant/home-assistant/pull/17920) will soon be merged.

File: source/_components/w800rf32.markdown [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+title: "W800rf32"
+description: "Instructions on how to integrate a W800rf32(a) into Home Assistant."
+date: 2018-10-16 10:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: w800rf32.png
+ha_category: Hub
+ha_release: 0.79
```
Comment (klaasnicolaas) [inline source/_components/w800rf32.markdown]: Same

File: source/_components/w800rf32.markdown (hunk 3 above)
Comment (klaasnicolaas) [reply source/_components/w800rf32.markdown]: This will fix the mapping value error

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by horga83** [comment]
Updated description header and added PR number from home-assistant.

**Comment by horga83** [comment]
This has been reviewed and approved by klassnicolass, is it possible to remove the block now?

File: source/_components/binary_sensor.w800rf32.markdown [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        name: motion_hall
+      a2:
+        name: motion_kitchen
+```
+
+{% configuration %}
+devices:
+  description: A list of devices.
+  required: true
+  type: list
```
Comment (MartinHjelmare) [inline source/_components/binary_sensor.w800rf32.markdown]: I think this should be `map`.

Comment (klaasnicolaas) [reply source/_components/binary_sensor.w800rf32.markdown]: I think that will not matter, with both `list` and `map` you will define keys.

**Comment by klaasnicolaas** [comment]
@horga83 What do you mean with the "blocks"?

**Review by MartinHjelmare — APPROVED** [review]
@klaasnicolaas ok with you?

**Review by klaasnicolaas — APPROVED** [review]
@MartinHjelmare Only thing I noticed is that some lines have been truncated with an enter. I do not think that is necessary, for the rest I approve of it.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
@klaasnicolaas In markdown, those enters are perfectly acceptable (some people even have the opinion you should use them, this includes myself...).

Currently, the documentation is mixed on this, so I'm fine with it.

**Pull request merged**